.gitignore

```diff
@@ -1,10 +1,23 @@
-/bin/
-/build*/
-/local*/
-/src/*_all.cpp
-
-**.pdb
-**.idb
-CMakeLists.txt.user
-.buildrc.cmake
-
+/bin/
+/build*/
+/local*/
+/src/*_all.cpp
+
+**.pdb
+**.idb
+CMakeLists.txt.user
+.buildrc.cmake
+**.dll
+**.zip
+/RUN2/

```

Comment:
i can live with filtering binary files ( dll, exe, zip ) ( ok, zip, wtf ), but the local folder names are no-no. actually I'd prefer to remove the existing /bin/ /build/ w/e as well.
You can specify per-project own excludes in the .git/info/exclude file the same way you'd do with .gitignore, with the added benefit of those not being committed.